### PR TITLE
Loader component

### PR DESCRIPTION
**Pull request by PawelNackowski**
Hi ! I created Loader :)

### CODE REVIEW

**Review by AllmostHumann — CHANGES_REQUESTED** [review]
I have no objections. 

Just let's use this convention for breakpoints ;)

`const mobileMinBp = ({ theme }) => theme.breakpoints.mobileMin;`

File: src/Common/Loader/styled.js
Comment (alajurczak) [inline src/Common/Loader/styled.js:1]: I think "index.js" should also appear in the Loader folder. In index.js also add e.g. "Header" for displaying "Search results for "Mulan".

File: src/Common/Loader/styled.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  animation: spin 1s linear infinite;
+    @keyframes spin {
+      0% {
+        transform: rotate(0deg);
+      };
+      100% {
+        transform: rotate(360deg);
+      };
+  };
+     @media (max-width: ${({ theme }) => theme.breakpoints.mobileMin}px){
```
Comment (alajurczak) [inline src/Common/Loader/styled.js:19]: As Jakub said, it's worth using one convention about styling breakpoints.

**Review by alajurczak — CHANGES_REQUESTED** [review]
I have a few comments :)

File: src/Common/Loader/styled.js
Comment (PawelNackowski) [reply src/Common/Loader/styled.js:1]: I think Loader is a separate component that can be used for loading posters, movie page, loading page, and other places. What do you think?

File: src/Common/Loader/styled.js (hunk 1 above)
Comment (PawelNackowski) [reply src/Common/Loader/styled.js]: I forgot about that. I corrected it 😄

File: src/Common/Loader/styled.js
Comment (AllmostHumann) [reply src/Common/Loader/styled.js:1]: Generally yes "Loader" should be a separate component, but index.js and styled.js I would do separately ;).

In index.js you could make a normal arrow function and make a component consisting of container, title and spinner.

Out of curiosity, why didn't you download the spinner graphic from the figma project?

Comment (alajurczak) [reply src/Common/Loader/styled.js:1]: Yeah, that's what I meant. 
I think he did it right when it comes to spinner graphic :)

Comment (PawelNackowski) [reply src/Common/Loader/styled.js:1]: I found it easier to adapt when it was made from scratch in CSS. 😀

**Comment by PawelNackowski** [comment]
I created a loading page. Please check 😃

**Review by alajurczak — APPROVED** [review]
It looks greate :)

**Comment by AllmostHumann** [comment]
Nothing to add for me 😉

**Review by AllmostHumann — APPROVED** [review]
Looking good to me top 😁

**Pull request merged**